style.md

```diff
@@ -8,12 +8,12 @@ For example, this paragraph will span three lines in the Markdown source.
 
 ## <a name="styleHex" />Traditionally hex settings should use JSON integers, not JSON strings
 
-For example, [`"classID": 1048577`][class-id] instead of `"classID": "0x100001"`.
+For example, [`"classID": 1048577`](config-linux.md#network) instead of `"classID": "0x100001"`.
 The config JSON isn't enough of a UI to be worth jumping through string <-> integer hoops to support an 0x… form ([source][integer-over-hex]).
 
 ## <a name="styleConstantNames" />Constant names should keep redundant prefixes
 
-For example, `CAP_KILL` instead of `KILL` in [**`linux.capabilities`**][capabilities].
+For example, `CAP_KILL` instead of `KILL` in [**`process.capabilities`**](config.md#process).
```

Comment:
Shouldn't this be `config.md#configProcess`?

(`## <a name="configProcess" />Process` is the section header of `config.md` which mentions `capabilities`)